src/index.html

```diff
@@ -8,6 +8,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star">
+        <img class="image"
+          src="./images/star.svg"
+          alt="star1"
+        >
+      </div>
```

Comment:
all stars should be just divs, not images, add image with css

```suggestion
      <div class="star"></div>
```

Reply:
also add corresponding classes to these elements (read task carefully)

```
Implement 6 blocks with stars class
The first should have stars--0 class
The next 5 blocks should have an extra class stars--1, stars--2 ... stars--5 (one extra class per block)
```